Tribler/community/tunnel/tunnel_community.py

```diff
@@ -184,7 +186,16 @@ def __init__(self, install_dir=None):
         self.max_traffic = 55 * 1024 * 1024
 
         self.max_packets_without_reply = 50
-
+        
+        if tribler_session:
+            state_dir = tribler_session.get_state_dir()
```

Comment:
You're right! I already thought that this should be much easier, but first did not work (got the error that this method was not existing in class Session), but after manual removing of the compiled python files it worked... :+1: